cypress/integration/pages/frontPage/tests.js

```diff
@@ -175,8 +175,28 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
               const usefulLinks = pageData.find(data => {
                 return data.type === 'useful-links';
               });
+
+              const contentTypes = [
+                'Text',
+                'Feature',
+                'Audio',
+                'Video',
+                'Gallery',
+                'Guide',
+              ];
+
+              const usefulLinksItems = usefulLinks.items.filter(item => {
+                return (
+                  item.assetTypeCode === 'PRO' &&
+                  contentTypes.includes(item.contentType)
+                );
+              });
+
               const isValidUsefulLinks =
-                usefulLinks && 'strapline' in usefulLinks;
+                usefulLinks &&
+                'strapline' in usefulLinks &&
+                Array.isArray(usefulLinksItems) &&
```

Comment:
We don't need this extra check here since  we have already assumed on line `88` that `usefulLinks.items` is an array by applying filter on the items.